sencha-workspace/SlateAdmin/app/model/person/Relationship.js

```diff
@@ -3,11 +3,14 @@ Ext.define('SlateAdmin.model.person.Relationship', {
     extend: 'Ext.data.Model',
     requires: [
 //        'Jarvus.ext.override.data.CallbackValidation',
+        'SlateAdmin.model.validator.CallbackValidator',
         'SlateAdmin.proxy.Records'
     ],
 
     idProperty: 'ID',
 
+    validationSeparator: 'jjjjjjjjjjjjj',
```

Comment:
yuck, do we need this? I don't think we need to store multiple validation errors per field, the default behavior of only storing the first seems like what we want and have relied on in the past